frontend/src/router/index.ts

```diff
@@ -72,6 +75,13 @@ const router = createRouter({
         isPublic: true
       }
     },
+    {
+      path: GEM_LOGS_ROUTE,
+      component: GemLogs,
+      meta: {
+        isPublic: true
+      }
```

Comment:
this should probably not be public? If my understanding is correct, public means accessible without being logged in